roles/swift-proxy/templates/etc/swift/proxy-server.conf

```diff
@@ -85,8 +85,10 @@ use = egg:swift#dlo
 
 [filter:authtoken]
 paste.filter_factory = keystonemiddleware.auth_token:filter_factory
-identity_uri = https://{{ endpoints.keystone }}:35358/
-auth_uri = https://{{ endpoints.keystone }}:5001/
+identity_uri = {{ endpoints.keystone.url.admin }}/
+auth_uri = {{ endpoints.keystone.url.internal }}/
+auth_host = {{ endpoints.main }}
+auth_port = {{ endpoints.keystone.port.haproxy_api }}
```

Comment:
Is auth_host and auth_port necessary here? I thought keystonemiddleware could get all it needs from identity_uri and auth_uri.